src/style.css

```diff
@@ -0,0 +1,105 @@
+:root {
+  --fontColore: #3d4e61;
+  --shadowColore: #3d4e611a;
+  --borderColore: #e1e7ed;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
```

Comment:
The wildcard selector is better to use for the box-sizing fix, not for resetting margins, padding, or borders.